rust/openvasd/src/controller/entry.rs

```diff
@@ -185,10 +185,6 @@ where
         let cid = self.cid.clone();
         Box::pin(async move {
             use KnownPaths::*;
-            // on head requests we just return an empty response without checking the api key
```

Comment:
I think that effectively disables HEAD for all endpoints but for `/scans`.

Reply:
Fixed. It behaves as before, except for /scans, which checks for authentication. It responds with 204 or 401 given the case.